Gemfile.lock

```diff
@@ -185,12 +185,12 @@ GEM
     factory_bot_rails (6.2.0)
       factory_bot (~> 6.2.0)
       railties (>= 5.0.0)
-    faker (2.23.0)
+    faker (3.0.0)
```

Comment:
There's an API change here, but a quick grep shows we don't use the old API in framework - https://github.com/faker-ruby/faker/blob/0ebdde5f876e942bc31a80c728a16774c8596ec5/CHANGELOG.md#breaking-changes